src/ASM/LR/ASMu2D.C

```diff
@@ -2180,10 +2180,16 @@ bool ASMu2D::matchNeighbour (ASMunstruct* neigh, int midx, int sidx, int orient)
 }
 
 
-void ASMu2D::remapErrors (RealArray& errors, const RealArray& origErr) const
+void ASMu2D::remapErrors (RealArray& errors,
+                          const RealArray& origErr, bool elemErrors) const
 {
   const LR::LRSplineSurface* basis = this->getBasis(1);
 
+  if (elemErrors) {
+    errors = origErr;
+    return;
+  }
+
```

Comment:
I wrote _before_ line 2186 (now the basis variable is unused of elemErrors is true, but never mind. Some other time (also for the ASUm3D)

Reply:
woops. missed that bit. you are ofc right.